Example/Tests/Matchers/CDTQQueryMatcher.h

```diff
@@ -0,0 +1,15 @@
+//
+//  CDTQQueryMatcher.h
+//  CloudantQueryObjc
+//
+//  Created by Rhys Short on 21/01/2015.
+//  Copyright (c) 2015 Michael Rhodes. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "Expecta.h"
+
+EXPMatcherInterface(beTheSameAs, (NSDictionary * expected));
+
+#define beTheSameQueryAs beTheSameAs
```

Comment:
I don't think we should have both of these defined. I suggest just keeping `beTheSameQueryAs` to keep the matcher's job more tightly specified.